docs/README.md

```diff
@@ -1,23 +1,28 @@
-..
```

Comment:
Is this added by the license plugin or added manually? If it's from license plugin, I think that we should fix the issue.

Reply:
I don't think so. Its likely that I copied and edited the file manually. Let me check the history.